API/Models/DTOmodels/ExamRequestDto.cs

```diff
@@ -3,12 +3,14 @@
     public class ExamRequestDto
     {
         public int Id { get; set; }
+        public int CourseId { get; set; }
         public string CourseName { get; set; }
         public string FirstNameProf { get; set; }
         public string LastNameProf { get; set; }
         public DateTime ExamDate { get; set; }
         public TimeSpan TimeStart { get; set; }
         public string Status { get; set; }
+        public string Details { get; set; }
```

Comment:
Details? Te referi la **description**?

Reply:
Nu, la detaliile pe care studentii ii lasa cand creeaza un exam request

Reply:
a e ok, am uitat ca aveam support in baza de date pentru asta
![image](https://github.com/user-attachments/assets/3575ab54-f24d-44e3-83b7-bb6ed5039fe0)